vql/networking/http_client.go

```diff
@@ -51,6 +51,14 @@ var (
 	mu sync.Mutex
 
 	proxyHandler = http.ProxyFromEnvironment
+
+	validHttpMethods = map[string]bool{
```

Comment:
this one disappeared no? maybe you need to rebase?

Reply:
Yeah, thought I did. Weird